keymaps/ava.json

```diff
@@ -1,6 +1,7 @@
 {
 	"atom-workspace": {
-		"ctrl-alt-r": "ava:run",
-		"ctrl-alt-a": "ava:toggle"
+		"ctrl-alt-shift-a": "ava:run",
+		"ctrl-alt-a": "ava:run-all",
+		"ctrl-alt-t": "ava:toggle"
```

Comment:
Researching about this. Having some problems to detect when the pane is active. Also, the part of the application that is usually active when running tests is the editor.

I'm trying to figure this out.